apigateway/templates/sign_cookies_api.json.tpl

```diff
@@ -0,0 +1,100 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "version" : "2021-09-06T09:43:11Z",
```

Comment:
Does it matter the version is a timestamp? Was this something auto-generated?

Reply:
It is auto-generated, how did you guess :smile: It should be a semantic version according [to the spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.2.md#versions) so I've set it to `0.0.1`